CMakeLists.txt

```diff
@@ -13,7 +13,23 @@ option(Picasso_REQUIRE_ARBORX "Build with ArborX support" OFF)
 set(CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake ${CMAKE_MODULE_PATH})
 find_package(CLANG_FORMAT 10)
 find_package(Cabana REQUIRED 0.5 COMPONENTS Cabana::Grid Cabana::Core)
-find_package(Boost 1.66.0 REQUIRED)
+
+# Download and install json if not found (input parsing)
+if (${CMAKE_VERSION} VERSION_GREATER_EQUAL "3.24")
+  cmake_policy(SET CMP0135 NEW)
+endif()
+find_package(nlohmann_json 3.10.0 QUIET)
+if(NOT NLOHMANN_JSON_FOUND)
+  include(FetchContent)
+  # Using most recent release here
+  FetchContent_Declare(json URL https://github.com/nlohmann/json/releases/download/v3.11.3/json.tar.xz)
```

Comment:
Is there a latest url not tied to a specific version here?

Reply:
Yeah, we could do `master` instead

Reply:
Let's be consistent with the version used in the CI.yml (3.10.0) then. Targeting `develop` in the nightly seems fine as it is.

Reply:
That's just the min version, so unless you disagree I'll leave it as is